bluemira/equilibria/equilibrium.py

```diff
@@ -227,11 +227,13 @@ def _get_eqdsk(cls, filename, force_symmetry=False):
             psi = e["psi"]
             e["dxc"] = e["dxc"] / 2
             e["dzc"] = e["dzc"] / 2
+        elif "fiesta" in e["name"].lower():
```

Comment:
Can we combine this with the first if statement as they do the same thing?

Reply:
I'm not sure that this is the only difference, or that SCENE and FIESTA have the same convention, so I've left room for treating different codes (with different conventions) differently.